### ISSUE TITLE
[ObjectStatus]: [a11y] Interactive object status announcement announces button more than once

### ISSUE THREAD

**Issue by lateefsofi** [SAP/ui5-webcomponents-react#7039]
### Describe the bug

We have an a11y for one of our usages of ObjectStatus component
Issue: Object status text is announced with 2 times button announce 
As per our internal a11y defect, button at the end in the announcement is not needed.
![Image](https://github.com/user-attachments/assets/c34aeaaf-f08d-431e-9a4f-ec3a9d9169c3)

### Isolated Example

https://stackblitz.com/edit/github-xwlddibr?file=src%2FApp.tsx

### Reproduction steps

1. Navigate to https://stackblitz.com/edit/github-xwlddibr?file=src%2FApp.tsx
2. Turn on screen reader
3. Focus to Object status
4. Check the announcement
5. 
...

### Expected Behaviour

While announcing object status text, button should be announced only once.

### UI5 Web Components for React Version

2.7.2

### UI5 Web Components Version

2.7.3

### Browser

Chrome

### Relevant log output

```shell

```

### Organization

Successfactors

### Declaration

- [x] I’m not disclosing any internal or sensitive information.
